app/modules/sightings/models.py

```diff
@@ -18,6 +19,13 @@ class SightingAssets(db.Model, HoustonModel):
     asset = db.relationship('Asset')
 
 
+class SightingStage(str, enum.Enum):
+    identification = 'identification'
+    un_reviewed = 'un-reviewed'
```

Comment:
Is an underscore for the variable and a hyphen for the string value a pattern we have established already? I'd have a mild preference to have them both be the same with an underscore.